prow/cmd/line/main.go

```diff
@@ -339,7 +340,7 @@ func (c *testClient) guberURL(build string) string {
 	return fmt.Sprintf("%s/%s/%s/%s/", url, prName, c.Job.Name, build)
 }
 
-func (c *testClient) tryCreateStatus(state, desc, url string) {
+func (c *testClient) tryCreateStatus(podName, state, desc, url string) {
```

Comment:
I guess, but the `try` reflects the fact that I'm not propagating up any errors and it might fail.